static/templates/user_presence_row.handlebars

```diff
@@ -3,7 +3,7 @@
     <span class="user-status-indicator"></span>
     <span class="user-device-indicator">{{#if mobile}}{{/if}}</span>
       <a href="#" data-email="{{email}}" data-name="{{name}}" title="{{name}} {{type_desc}}"
-         class="{{#if my_fullname}} my_fullname{{/if}}">{{name}}</a>
+        class="{{#if my_fullname}} my_fullname{{/if}}">{{name}}{{#if starred}}(<span class="icon-vector-star"></span>){{/if}}</a>
```

Comment:
@timabbott do you have thoughts on the UI here? Maybe instead of starring the users, we just have "pinned users", and a `<hr>` separating the pinned users from the rest. Or maybe we turn their presence circle into a presence star? Right now this seems clunky.

@krtkmj: for now, at least right-align the icon. You can accomplish this with another `<div>`